### DISCUSSION TITLE
特定のキーが反応しない

### DISCUSSION THREAD

**Discussion by KazukiNagao** [Yowkees/keyball#411] [Q&A]
Keyball44にて、以下画像のキーのみどうしても反応しませんでした。
回路図も過去に公開されてるようですが、見方が分からず…どこのキーソケットとダイオードを見直せばよいでしょうか…？

![IMG_1985](https://github.com/Yowkees/keyball/assets/5966602/bf7df900-f453-4e41-8e77-0930e4b3f162)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

写真の色同士のキーソケットとダイオードがそれぞれ対応していますので、ご確認をよろしくお願いいたします。
キーソケットのはんだ付けが原因の場合もありますので、キーソケットのはんだ付けの見直しも併せて行っていただけますと幸いです。
![280289126-bf7df900-f453-4e41-8e77-0930e4b3f162 (1)](https://github.com/Yowkees/keyball/assets/143698349/620c84c7-4476-42e4-a204-d2794b062c93)

**Reply by curuknagao** [reply]
ありがとうございます！
無事キー反応できました！

**Reply by Ryo624** [reply]
無事に解決していただきましてありがとうございます！
ぜひ末永くご愛用していただければ幸いです。
